### PR TITLE
documents: add and improve fields on detailed view

### LINKED ISSUE #723: Improvements and corrections of the document detailed view [3]
The following fields are displayed on the document detailed view (public + professional). See mockups below for detailed display specifications. The display sequence is as indicated in the mockup

Fields:
* `notes`
    * Currently not displayed > to display
    * Sequence of the values should respect the order in the JSON array
* `contentNote`
    * Currently not displayed > to display
    * Label: "Content"
    * If repeated, each note on a new line
* `editionStatement`
    * Currently not displayed > to display
    * This fields value is displayed without label (as in the mockup)
* `contributions`
    * Currently displayed > but need to distinguish agents of type `bf:Meeting` from the others
    * if `contribution.agent.type` is "bf:Meeting"
        * display below Language (see mockup)
        * Template: `preferred_name` (`number` : `date` : `place`)
        * Label: Conference
        * Each contribution on a new line
    * if `contribution.agent.type` is NOT "bf:Meeting"
        * keep the current display below the title
    * example : https://doc.rero.ch/record/329465 / https://sonar.test.rero.ch/usi/documents/4315
* `partOf`
    * Currently displayed, but some sub-fields are missing 
    * `document.title` / `document.contribution[1]` ; `document.contribution[x]`. - `document.publication.statement`. - `numberingYear`, vol. `numberingVolume`, no. `numberingIssue`, p. `numberingPages`
    * ideally, the abbreviations "vol.", "no." and "p." are translated
* `series`
    * Currently not displayed > to display
    * Label "Series statement"
    * Subfields title and numbering of series are separated by ` ; ` (space, semicolon, space)
    * If repeated, each seriesStatement on a new line
* `additionalMaterials`
    * Currently not displayed > to display
    * Label: Accompanying material
* `otherMaterialCharacteristics`
    * Currently not displayed > to display
    * Label: Other material characteristics
* `oa_status`
    * Currently not displayed > to display
    * Label: Open Access status

[MOCKUP: search_v5.pdf](https://github.com/rero/sonar/files/8254633/search_v5.pdf)

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Adds fields: notes, contentNote, editionStatement, series,
  additionalMaterials, otherMaterialCharacteristics, oa_status.
* Splits the data in the contributions field (meeting and other).
* Improves formatting on the partOf field.
* Closes https://github.com/rero/sonar/issues/723.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Dependency

Sonar ui: https://github.com/rero/sonar-ui/pull/303

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: sonar/modules/documents/views.py [hunk 1]
```diff
@@ -170,36 +170,71 @@ def part_of_format(part_of):
...
 
 
 @blueprint.app_template_filter()
-def contributors(record):
+def contributors(record, meeting=False):
     """Get ordered list of contributors."""
     if not record.get('contribution'):
         return []
 
+    print('>>>>> CONT: ', record.get('contribution'))
```
Comment (BadrAly) [inline sonar/modules/documents/views.py:226]: remove the `print` line

**Review by BadrAly — APPROVED** [review]

File: sonar/modules/documents/views.py [hunk 2]
```diff
@@ -170,36 +170,70 @@ def part_of_format(part_of):
...
-        items.append(part_of['document']['title'])
-
+    contributions = []
+    document = part_of.get('document', {})
+    if document.get('title'):
+        items.append(document['title'])
+    if 'contribution' in document:
+        contribution = document['contribution'].pop(0);
+        if len(document['contribution']) > 0:
+            contributions.append(' / {value} ; '.format(value=contribution))
```
Comment (jma) [inline sonar/modules/documents/views.py:180]: Can be done by a `join`

**Review by jma — APPROVED** [review]
Can you put more description in your commit message?

**Comment by PascalRepond** [comment]
- [ ] REP: Open Access status should be just above Identifiers (see mockup)

**Review by PascalRepond — APPROVED** [review]

**Pull request merged**